### PR TITLE
Fix `gitleaks` issue

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: .github/workflows/main_workflow.yml [hunk 1]
```diff
@@ -161,7 +161,7 @@ jobs:
           wget https://github.com/zricethezav/gitleaks/releases/download/v8.10.1/gitleaks_8.10.1_linux_x64.tar.gz && \
           tar -xzf gitleaks_8.10.1_linux_x64.tar.gz && \
           sudo install gitleaks /usr/bin && \
-          gitleaks detect --report-format json --report-path leak_report
+          gitleaks detect --report-format json --report-path leak_report -v
```
Comment (angrybayblade) [inline .github/workflows/main_workflow.yml:164]: Prints the report in the CI

Comment (angrybayblade) [inline .github/workflows/main_workflow.yml:164]: Prints the report in the CI

File: .gitleaks.toml [hunk 2]
```diff
@@ -10,6 +10,7 @@ regexes = [
     '''^([a-zA-Z_][a-zA-Z0-9_]{0,127})/([a-zA-Z_][a-zA-Z0-9_]{0,127})(:(any|latest|((0|[1-9]\\d*))\\.((0|[1-9]\\d*))\\.((0|[1-9]\\d*))(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?))?(:[a-zA-Z_][a-zA-Z0-9]{45})?$''',  # sourced from aea/configurations/data_types.py
     '''[a-zA-Z_][a-zA-Z0-9]{45}''',
     '''(Qm(.){44}\b)''',
+    '''(cross_period_persisted_keys_\d+)''',
```
Comment (angrybayblade) [inline .gitleaks.toml:13]: https://github.com/valory-xyz/open-autonomy/blob/ee71f2f577c7baa906508e4003c0b9090a3168b7/packages/valory/skills/abstract_round_abci/tests/test_abci_app_chain.py#L105-L106

File: .gitleaks.toml [hunk 3]
```diff
@@ -22,6 +23,7 @@ paths = [
     '''deployments/keys/cluster_ropsten_keys.json''',
     '''deployments/keys/hardhat_keys.json''',
     '''(^(.*)\/node_modules(.))''',
+    '''tests/data/logs/''',
```
Comment (angrybayblade) [inline .gitleaks.toml:26]: Introduced on this PR https://github.com/valory-xyz/open-autonomy/pull/1834

**Comment by angrybayblade** [comment]
Failure is caused by a connection timeout

**Review by Adamantios — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**